### PR TITLE
Adding Joss Paper Info

### PR DESCRIPTION
**Pull request by bzenger**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+assumed to be approximately periodic, e.g., a sequence of heart
+beats. This capability is novel and we know of no other published
+description that achieves these goals. The combination of flexibility,
+minimal computational requirements, time saving algorithms,
+and a structured framework easy to modify and extend makes PFEIFER the
+ideal toolkit for researchers processing cardiac electrical signals.
+
+
+---
+nocite: |
```
Comment (jessdtate) [inline paper.md]: I don't understand what this line is for. The formatting is messed up and they can be cited in the body of the text

File: paper.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
```
Comment (jessdtate) [inline paper.md]: This file and the bib file should go in a folder called josspaper, or something like that

File: paper.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+   index: 2
+date: 25 October 2017
+bibliography: josspaper.bib
+---
+
+# Summary
+
+##High Level Functionality
+
+Preprocessing Framework for Electrograms Intermittently Fiducialized from
```
Comment (jessdtate) [inline paper.md]: This is a confusing sentence

**Review by jessdtate — CHANGES_REQUESTED** [review]

File: paper.md (hunk 2 above)
Comment (bzenger) [reply paper.md]: I agree we could do it this way but several examples have also done it the way I did it here

File: paper.md (hunk 3 above)
Comment (bzenger) [reply paper.md]: It is the acronym for PFEIFER.

File: paper.md (hunk 1 above)
Comment (bzenger) [reply paper.md]: None of the references render in any of the papers submitted. Joss has to render them is some special formatting that does this for us.

File: paper.md (hunk 3 above)
Comment (jessdtate) [reply paper.md]: That is not the whole sentence

File: paper.md (hunk 1 above)
Comment (jessdtate) [reply paper.md]: none of the submitted papers have a line like this, even when they do not cite the references.  Is this something that was in the instructions?  it renders very badly.

File: paper.md (hunk 2 above)
Comment (jessdtate) [reply paper.md]: If we can do it either way, then we should put it in a folder.

**Review by jessdtate — APPROVED** [review]

**Pull request merged**